.github/workflows/acceptance-tests-runner.yml

```diff
@@ -306,7 +306,7 @@ jobs:
           terraform_wrapper: false  
       - name: Acceptance Tests
         env:
-          MONGODB_ATLAS_ENABLE_BETA: ${{ vars.MONGODB_ATLAS_ENABLE_BETA }}
+          MONGODB_ATLAS_ENABLE_PREVIEW: ${{ vars.MONGODB_ATLAS_ENABLE_PREVIEW }}
```

Comment:
vars.MONGODB_ATLAS_ENABLE_PREVIEW can be replaces by true as in mig config

Reply:
Sorry, I didn't see these changes before merging 🤦
Maybe we can clean it up in one of the test  improvement PRs?